bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -106,6 +106,7 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         {% endif -%}
         check_existence=True,
         mode='reschedule',
+        poke_interval=datetime.timedelta(minutes=5),
```

Comment:
This should be `poll_interval` ([docs](https://airflow.apache.org/docs/apache-airflow/2.9.3/_api/airflow/sensors/external_task/index.html#airflow.sensors.external_task.ExternalTaskSensor)):
```suggestion
        poll_interval=datetime.timedelta(minutes=5),
```
(ditto for the other `ExternalTaskSensor`s)

Reply:
Never mind. It looks like `poll_interval` only applies when the sensors are run in deferrable mode ([code](https://github.com/apache/airflow/blob/c92929b99fc15c42ccc0799cc696417a5af76161/airflow/sensors/external_task.py#L357)).